.github/workflows/buildAndTest.yml

```diff
@@ -6,19 +6,12 @@ on:
       - main
   pull_request:
   workflow_dispatch:
-    inputs:
-      release_id:
-        description: 'Release id to upload artifacts to'
-        default: ''
-      python_package_version:
-        description: 'Version to use for creating the Python package'
-        default: ''
 
 jobs:
   build:
     name: Build and Test (Release Asserts)
     # Changes to the name of this job needs to be synced with releaseSnapshotPackage.yml.
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
```

Comment:
Lets leave this as 20.04. You want to upgrade it on your own time, not because GitHub pushed a new release one morning!

Reply:
(and below)

Reply:
done